sway/commands/input/xkb_file.c

```diff
@@ -0,0 +1,22 @@
+#define _POSIX_C_SOURCE 200809L
+#include "sway/config.h"
+#include "sway/commands.h"
+#include "sway/input/input-manager.h"
+#include "log.h"
+
+struct cmd_results *input_cmd_xkb_file(int argc, char **argv) {
+	struct cmd_results *error = NULL;
+	if ((error = checkarg(argc, "xkb_file", EXPECTED_EQUAL_TO, 1))) {
+		return error;
+	}
+	struct input_config *ic = config->handler_context.input_config;
+	if (!ic) {
+		return cmd_results_new(CMD_FAILURE, "No input device defined.");
+	}
+
+	ic->xkb_file = strdup(argv[0]);
```

Comment:
There should be some way of resetting this to `NULL` so the rules can be used instead. We usually use `-` for that purpose

Reply:
The RMLVO commands, as well as this command, can only be run in a config file. This is the same exact format for a c file as the xkb_layout command. Is there still an issue here? Should all the RMLVO commands be changed in this way?

Reply:
> The RMLVO commands, as well as this command, can only be run in a config file.

You can change them at runtime with `swaymsg` (or any IPC client). The only input subcommands that are config only are `xkb_capslock` and `xkb_numlock`.

> Should all the RMLVO commands be changed in this way?

No, they are fine as-is, but `xkb_file` shadows them when set so there needs to be a way to unset it.

Reply:
Sounds good